samples/crypto/test_sample/README.rst

```diff
@@ -0,0 +1,258 @@
+.. _crypto_test:
+
+Cryptography test sample
+########################
+
+The cryptography test sample verifies the functionality of the nrf_crypto operations by using known test vectors approved by the National Institute of Standards and Technology (NIST) and others.
+All available nrf_crypto backends can be tested by modifying the nrf_crypto section in the sdk_config file, as described in lib_crypto_config.
```

Comment:
This line must be replaced with instructions refering to the nrf_security configuration along the lines of:

"The tests are executed if they cryptographic functionality is enabled in kconfig. Please configure nrf_security and all available hardware or software backends to enable the tests"

or something to that effect

Reply:
This has been resolved with a rewrite